vizro-core/src/vizro/models/_controls/filter.py

```diff
@@ -36,6 +36,18 @@
 }
 
 
+def _convert_to_date(value):
+    return pd.to_datetime(value, format="mixed")
+
+
+def _is_convertible_to_date(pd_series: pd.Series) -> bool:
```

Comment:
Just FYI because I think we should get rid of this, but this is a very well written function 👏